src/euphorie/client/model.py

```diff
@@ -566,9 +571,13 @@ def allowed(self, object, object_roles=None):
         if object_roles is None:
             return True
 
-        for role in ["Anonymous", "Authenticated", "EuphorieUser"]:
-            if role in object_roles:
-                return True
+        if not isinstance(object_roles, (list, tuple)):
```

Comment:
To my understanding `object_roles` should always be an iterable.
Is this check you added just needed to support the existing tests?
If yes please revert and modify the tests.